Client/WebFilters/AdblockResourcesMappings.swift

```diff
@@ -14,37 +21,45 @@ enum ResourceLocale: String {
     case ar, bg, zh, cs, de, da, et, fi, fr, el, hu, id, hi, fa, `is`, he, it, ja, ko, lt, lv, nl,
     pl, ru, uk, be, es, sl, sv, tr, vi
     
-    var resourceName: String {
+    func resourceName(for fileType: FileType) -> String {
+        var resourceId = ""
+        
         switch self {
```

Comment:
Do we prefer:
```
let map = [.ar: "...", .bg: "...", .zh: "..."]
resourceId = map[fileTypye] ?? ""
```
or `switch` statement instead?

It's just a question. You don't have to change your stuff.

Reply:
I don't see how this code would go then, `map` is a `locale:id`  dictionary , while `fileType` is a different enum. So I can't reallty do it like that